main.py

```diff
@@ -16,10 +17,20 @@
 screen = pygame.display.set_mode(size)
 GRAY = (150, 150, 150)
 auto = False
-board = Board(4, 4)
 animate_percentage = 0
 last_direction = 'up'
 
+#Board Logic
```

Comment:
Code style: Line comments should have a space after the pound sign, i.e.
`# Allow for...`